homework6.py

```diff
@@ -0,0 +1,47 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+# stats: кількість записів
+# add: додати запис
+# delete <name>: видалити запис за іменем (ключем)
+# list: список всіх імен в книзі
+# show <name>: детальна інформація по імені
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+
+my_contacts = {
+    'Wednesday Addams': '+442255667788',
+    'Gregory House': '+112233445566',
+    'Winston Churchill': '+445566778899'
+}
+
+command = input("Enter command (stats, add, delete, list, show): ")
+# format "command amount"
+
+if command == 'stats':
+    print(len(my_contacts))
+    # calculating the number of contacts
+
+elif command == 'add':
+    new_contact_name = input('Type name and surname: ')
+    new_contact_phone = input('Type phone number: ')
+    my_contacts[new_contact_name] = new_contact_phone
```

Comment:
You should check if the key already exists. It is not allowed to override records